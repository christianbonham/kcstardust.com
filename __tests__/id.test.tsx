import { render, screen } from '@testing-library/react';
import InstructionalDesign from '../pages/id';

describe('Instructional Design', () => {
  it('renders a heading', () => {
    render(<InstructionalDesign />);

    const heading = screen.getByRole('heading', {
      name: /Instructional Design Services/i,
    });

    expect(heading).toBeInTheDocument();
  });
});

it('renders Instruction Design Page unchanged', () => {
  const { container } = render(<InstructionalDesign />);
  expect(container).toMatchSnapshot();
});
