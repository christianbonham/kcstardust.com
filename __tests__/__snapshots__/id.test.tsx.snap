// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Instruction Design Page unchanged 1`] = `
<div>
  <div
    class="row "
  >
    <div
      class="margin null"
    />
    <div
      class="content null"
    >
      <div
        class="col"
      >
        <div
          class="mrq"
        >
          <h1
            class="title"
          >
            Instructional Design Services
          </h1>
          <p
            class="description"
          >
            We provide world-class instructional design and UX design and development services through all phases of development.
          </p>
        </div>
      </div>
    </div>
    <div
      class="margin null"
    />
  </div>
  <div
    class="row "
  >
    <div
      class="margin null"
    />
    <div
      class="content null"
    >
      <div
        class="col"
      >
        <div
          class="kcs"
        >
          <h1>
            Our Process
          </h1>
          <h2
            class="flair"
          >
            Engaging and enjoyable distance learning experiences for your team or business.
          </h2>
          <h2>
            Business Analysis and Training Consultation
          </h2>
          <p>
            Understand how team education and training can best be integrated in your business. Find out how distance learning can help you achieve your business goals.
          </p>
          <h2>
            Content development
          </h2>
          <p>
            Create engaging and effective learning materials and content. We can develop your education and training material from the ground up, or work with material you already have.
          </p>
          <h2>
            Test Driven Design and Development
          </h2>
          <p>
            In the best eLearning experiences, design and development continues after go-live. By tracking the behavior of learners through your courses, we can evaluate and implement design changes to improve the performance of your training.
          </p>
        </div>
      </div>
    </div>
    <div
      class="margin null"
    />
  </div>
</div>
`;
